services-directions-models/src/main/java/com/mapbox/api/directions/v5/utils/FormatUtils.java

```diff
@@ -0,0 +1,218 @@
+package com.mapbox.api.directions.v5.utils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.mapbox.geojson.Point;
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Methods to convert models to Strings.
+ */
+public class FormatUtils {
+  /**
+   * Returns a string containing the tokens joined by delimiters.
+   *
+   * @param delimiter the delimeter on which to split.
+   * @param tokens A list of objects to be joined.. Strings will be formed from the objects by
+   *               calling object.toString().
+   * @return {@link String}
+   */
+  public static @Nullable String join(@NonNull CharSequence delimiter, @Nullable List<?> tokens) {
```

Comment:
NIT
```suggestion
  @Nullable
  public static String join(@NonNull CharSequence delimiter, @Nullable List<?> tokens) {
```

Same thing applies to the rest of the methods 👇

Reply:
fixed